sys/riotboot/hdr.c

```diff
@@ -53,15 +53,15 @@ void riotboot_hdr_print(const riotboot_hdr_t *riotboot_hdr)
 int riotboot_hdr_validate(const riotboot_hdr_t *riotboot_hdr)
 {
     if (riotboot_hdr->magic_number != RIOTBOOT_MAGIC) {
-        LOG_INFO("%s: riotboot_hdr magic number invalid\n", __func__);
+        LOG_DEBUG("%s: riotboot_hdr magic number invalid\n", __func__);
```

Comment:
LOG_DEBUG needs to be defined in the fallback case further up the file.

Reply:
Thanks!